src/docs/development/platform-integration/add-to-app-android/add-flutter-screen.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Adding a Flutter screen to an Android app
+short-title: Add a Flutter screen
+description: Learn how to add a single Flutter screen to your existing Android app.
+---
+
+This guide describes how to add a single Flutter screen to an existing Android app. A Flutter
+screen can be added as a normal, opaque screen, or as a see-through, translucent screen. Both
+options are described in this guide.
+
+## Add a normal Flutter screen
+
+{% asset development/platform-integration/add-to-app-android/add-flutter-screen/add-single-flutter-screen_header.png class="mw-100" alt="Add Flutter Screen Header" %}
+
+### Step 1: Add Gradle dependencies
+
+TODO: update this with gradle instructions when the embedding is broken into multiple maven artifacts.
+
+### Step 2: Initialize Flutter
+
+Flutter requires that it be initialized before attempting to display any Flutter content. This can be done in one of three ways: 
```

Comment:
GLOBAL: We follow the GitHub convention of breaking line length at 80 chars or less. Often, we do it on a phrase..